test/functional/sqlsrv/sqlsrv_batch_query.phpt

```diff
@@ -0,0 +1,179 @@
+--TEST--
+Test a batch query with different cursor types
+--DESCRIPTION--
+Verifies that batch queries don't work with dynamic, static, and keyset
+server-side cursors, and checks that correct column and row counts are
+returned otherwise
+--SKIPIF--
```

Comment:
Please include the ODBC reference page(s) to this test that provide(s) reasons why batch queries will return errors with server-side cursors